### DISCUSSION TITLE
【任务】Solidity题库整理｜20,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#154] [任务 | Bounty]
目前WTF Solidity极简教程有40讲了，也有很多同学通过这个教程学习，但目前缺少方法检验自己的掌握程度。这个任务是为了WTF Solidity教程制作题库，检验学习者的掌握程度。

**任务**：WTF Solidity教程1-40讲每一讲的试题（形式包括：单选、多选、填空、判断），每5讲的小测（形式包括：单选、多选、填空、判断、程序补全），入门（1-15讲）、进阶（16-30讲）、应用（31讲之后）的大测验（形式包括：单选、多选、填空、判断、程序补全、合约编写）

**赏金**：20000 $PEOPLE 代币，根据工作量（最终选入题库的题数+参与组织）分配。

**规则**：每一讲的试题归总到 [Topics](https://github.com/AmazingAng/WTFSolidity/tree/main/Topics/Exams)

文件夹命名规则：
- 每一讲的随堂检验，以第一讲为例：`Quiz_01_HelloWeb3`
- 每5讲的小测，以第1-5讲小测为例：`Exam_01_05`
- 入门、进阶、应用的大测验，入门为`Exam_Baic`，进阶为`Exam_Advanced`、应用为`Exam_Application`

回帖参与任务，需要选出3个组织者，分别组织随堂小测，小测验，大测验。回帖时候注明是否想当组织者。

报名情况（组织者也可以编写）：
- 组织者：@0xc25fee20 @JustinAsdz
- 编写者：@AmazingAng @seasidejuvenile666 @vecpeng

**Comment by 0xc25fee20** [comment]
报名一个小测组织者

**Reply by AmazingAng** [reply]
好的

**Comment by seasidejuvenile666** [comment]
报名一个试题编写人员

**Reply by AmazingAng** [reply]
好的

**Comment by vecpeng** [comment]
报名一个随堂小测编写人员

**Comment by JustinAsdz** [comment]
随堂小测组织

**Comment by bapp-prog** [comment]
报名一个试题编写人员

**Reply by AmazingAng** [reply]
介绍一下你自己？

**Comment by 0xxfu** [comment]
大佬，我提交了（WTF技术交流微信群）表单，我可以做小测验编写者，组织者暂时还没有那么多精力

**Reply by AmazingAng** [reply]
好的

**Comment by cheng521521** [comment]
报名 入门的前5讲

**Reply by AmazingAng** [reply]
不按讲来算，等明天先讨论一下怎么整理。

**Comment by wyf-ACCEPT** [comment]
报名小测验编写者 或大测验编写者🙋‍♀️
技能点：【Solidity】【精通Python】【精通链上数据分析】【熟悉各类Defi和跨链项目】
来自 THUBA DAO

**Comment by hyc6446** [comment]
想报名 随堂小测，小测验编写人员。贡献量可能不是很多不知道可不可以。

**Reply by AmazingAng** [reply]
微信多少？

**Comment by Yinboan** [comment]
报名随堂小测编写（节数不太敢确定，至少一节哈哈哈）

**Comment by jokereven** [comment]
还缺人吗?
